bundles/org.openhab.binding.goecharger/src/main/java/org/openhab/binding/goecharger/internal/handler/GoEChargerHandler.java

```diff
@@ -299,29 +299,31 @@ private String getWriteUrl(String key, String value) {
 
     private void sendData(String key, String value) {
         String urlStr = getWriteUrl(key, value);
-        logger.debug("POST URL = {}", urlStr);
+        logger.trace("POST URL = {}", urlStr);
 
         try {
             HttpMethod httpMethod = HttpMethod.POST;
             ContentResponse contentResponse = httpClient.newRequest(urlStr).method(httpMethod)
                     .timeout(5, TimeUnit.SECONDS).send();
             String response = contentResponse.getContentAsString();
 
-            logger.debug("{} Response: {}", httpMethod.toString(), response);
+            logger.trace("{} Response: {}", httpMethod.toString(), response);
 
             var statusCode = contentResponse.getStatus();
             if (!(statusCode == 200 || statusCode == 204)) {
-                logger.warn("Could not send data, Response {}, StatusCode: {}", response, statusCode);
+                updateStatus(ThingStatus.OFFLINE);
```

Comment:
This method has some overloads, so that you can add a statusdetail to let the user know why it is offline.